openslides_backend/presenter/__init__.py

```diff
@@ -1 +1,3 @@
 from .presenter_interface import *  # noqa
+from . import initial_data  # noqa
```

Comment:
I strongly prefer that the init does not import more than the inferface file. As I did it for action module.

Reply:
Of course we can discuss this. My issue was that one may import the interface without "loading" all the stuff.

Reply:
> My issue was that one may import the interface without "loading" all the stuff.
May you elaborate on this one? What are the reasons behind this? Are there performance reasons?

Reply:
Lets make an extra issue for that.